site/using-cluster-operator.md

```diff
@@ -0,0 +1,790 @@
+# Using Kubernetes Cluster Operator
+
+This section covers how to deploy Custom Resource objects that will
+be managed by the Kubernetes Cluster Operator. If you have not installed
+the Cluster Operator, check the [Installing Cluster Operator in a Kubernetes cluster](/install-cluster-operator.html).
+
+## Confirm Service Availability
+
+Before configuring your app to use Cluster Operator, ensure that RabbitmqCluster Custom Resource is deployed
+to your Kubernetes cluster and is available.
+
+To confirm this availability:
+
+1. Run:
+
+    ```
+    kubectl get customresourcedefinitions.apiextensions.k8s.io
+    ```
+
+1. Verify that `rabbitmqclusters.rabbitmq.com` is on the list, as in the example below.
+
+    <pre class="terminal">
```

Comment:
Is `terminal` a [language Highlight.js supports](https://highlightjs.org/static/demo/)? I am only aware of Bash (`class="lang-bash"` in our case) and PowerShell (`lang-powershell`).

If `terminal` is a language name supported by Highlight.js we'd need to update our custom highlight.js build to include it.

Reply:
I've replaced the `class="terminal"` by `lang-bash` or `lang-yaml` where applicable. I will revisit other guides as well to make sure I'm using the correct classes 👌